src/UIComponents/ModelCard.jsx

```diff
@@ -45,52 +45,63 @@ class ModelCard extends Component {
         <div style={styles.modelCardContainer}>
           <h3 style={styles.modelCardHeader}>{trainedModelDetails.name}</h3>
           <div style={styles.modelCardSubpanel}>
-            <h5 style={styles.modelCardHeading}>Summary</h5>
+            <h5 style={styles.modelCardHeading}>Accuracy</h5>
             <div style={styles.modelCardContent}>
               <p style={styles.modelCardDetails}>
-                Predict {label.id} based on{" "}
-                {selectedFeatures.length > 0 && percentCorrect && (
+                {percentCorrect && (
```

Comment:
What happens if it's `0%`?  Is there a chance we want to show it but it won't show?

Reply:
![Screen Shot 2021-06-29 at 9 09 47 PM](https://user-images.githubusercontent.com/12300669/123887017-b262d580-d91e-11eb-8f71-3250a3848be9.png)

This is kind of interesting... in the case where `percentCorrect` === 0 the zero will render, but not 0%, which I think is probably fine. It's rare to have a 0% accurate model, and even more rare that someone would want to save a 0% accurate model because it'd be useless. _But_ I also trust that this component will always have `percentCorrect` when needed so it seems ok to remove the check for `percentCorrect` and just render the value.